### ISSUE TITLE
High contrast: find matches do not look high contrast friendly

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3288]
#3248 
I would expect the contrast to be better between the light orange and the black background
 
![screen shot 2016-02-23 at 11 09 15](https://cloud.githubusercontent.com/assets/1926584/13248325/ee91d408-da1d-11e5-9729-9ff045fa70d8.png)
